core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -152,6 +152,8 @@
     private Duration retryInitialDelay = new Duration(10, SECONDS);
```

Comment:
Please update commit message to:
```
Allow to hide inaccessible columns

With hide-inaccessible-columns configuration toggle now it is 
possible to exclude non accessible from  SELECT * statement
```

> It solves issue: #7461

let's keep this as part of PR description.